locale/es_ES.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -254,7 +254,7 @@ msgstr ""
 
 #: index.php:1233
 msgid "Windows:"
-msgstr ""
+msgstr "Windonws:"
```

Comment:
This looks like a typo.

Reply:
@stweil yes, it is fixed by the new commit